modules/persistent-storage-csi-snapshots-create.adoc

```diff
@@ -15,9 +15,9 @@ When you create a `VolumeSnapshot` object, {product-title} creates a volume snap
 * A storage class to provision the storage back end.
 * No pods are using the persistent volume claim (PVC) that you want to take a snapshot of.
 +
-[NOTE]
+[IMPORTANT]
```

Comment:
```suggestion
[WARNING]
```
- Per [our SSG guidelines](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions), I think this needs to be a WARNING admonition, which should be used for "Information about potential system damage, data loss, or a support-related issue if the user disregards this admonition." 

Can you please change this, and then ping me again for a merge? Thanks!

Reply:
Updated as per the suggestion.